[KAN-99] 메시지 스크린 레이아웃

k-cool opened the pull request:
# Pull Request

## 구현 화면

![ChattingListScreen](https://github.com/watermelon-blossom/blossom-FE/assets/39205612/a87de9f3-6e38-46e6-9ef9-30934da5d02f)

## 구현 기능

- 메시지 스크린 레이아웃

## 기타 질문 및 특이 사항

- TextInput 아이콘 적용가능하게 수정
- Icon 폴더명 수정
- createSvgIndex 스크립트 작성
- svg 파일명 스네이크 케이스 통일

## PR을 올렸다면 아래 사항은 반드시 지켜주세요.

- [x] JIRA Issues에서 적절한 이슈 하나를 연동하였습니다.
- [x] 본 PR을 팀원들과 사전에 상의하였고 팀원들 모두 해당 PR에 대하여 알고 있습니다.
- [x] 제가 의도한 파일들과 수정 사항들만 커밋이 된 것을 확인 하였습니다.
- [x] 기능이 제대로 실행되는지 확인 하였습니다.
- [x] yarn start로 구현한 화면 혹은 기능을 확인할 수 있습니다.
- [x] self-refactoring으로 WaterMelon FE 코딩 컨벤션에 준수하는지 점검하였습니다.

> 마크다운에서 체크를 하기 위해서는 [x] 이렇게 수정하시면 됩니다. [x ], [ x ] 가 아니라 정확하게 [x]여야 합니다. 혹은 마우스 클릭으로 체크해주세요.

k-cool committed:
[KAN-99] 메시지 스크린 레이아웃

- 메시지 스크린 레이아웃
- TextInput 아이콘 적용가능하게 수정
- Icon 폴더명 수정
- createSvgIndex 스크립트 작성
- svg 파일명 스네이크 케이스 통일

starz-woo commented on scripts/createSvgIndex.js:
```diff
@@ -0,0 +1,22 @@
+const { cwd } = require("process");
+
+const svgDirPath = path.join(cwd(), "/assets/icons");
+const indexPath = path.join(cwd(), "/assets/icons/index.ts");
```
⚠️script 만드시느라 icons로 폴더명을 정하신건가요? pull 받아서 실행할 때는 폴더 명이 Icons라 에러로 잡혀서 말씀드립니다! 아니면 폴더명을 icons로 변경부탁드립니다!

k-cool replied:
오잉? 저 수정해서 지금도 로컬에 icons 라고 되어있는데 풀 받았을때 반영이 안되던가요??? 
이게 깃이 대소문자 구분을 못하게 설정이 되어있을수도 있을것 같아요... 로컬 깃 설정을 수정해야할수도 있을것 같습니다. 한번 찾아볼게요!

<img width="324" alt="스크린샷 2024-07-12 오후 8 31 41" src="https://github.com/user-attachments/assets/2b7d5eba-97f1-4374-9dca-9b5a2bda2de4">

starz-woo reviewed (approved):
고생하셨습니다! 
몇 가지 수정이 되었으면 하는 것들이 있는데,
우선 메세지 리스트를 올릴 때 검색창과 스크롤되는 부분이 너무 붙어있어서 조금 간격을 설정하면 좋을 것 같습니다! 
그리고 검색창에 입력하려고 할 때 입력 시작 인식이 Search 단어 부분만 되는 것 같아서 이 부분이 수정이 가능하면 수정 부탁드립니다!
마지막으로 직접 핸드폰으로 실행해서 검색어를 입력하려고 할 때 키보드가 나오면서 하단 탭이 올라오는데 이 부분이 조금 이질적으로 느껴지긴 합니다. 이거에 관해서는 일요일 회의 때 한 번 얘기해보면 좋을 것 같습니다! 
선승인 드릴테니 가능한 것만 수정해주시고 머지 부탁드립니다~

k-cool committed:
[KAN-99] 메시지 스크린 레이아웃

- icons 폴더명 변경(깃 설정 변경)
- 키보드 띄울때 탭메뉴 사라지도록 수정
- input 아무곳을 터치해도 인식되도록 수정
- 메시지 스크린 레이아웃 조정
